### PR TITLE
add travis config to automatically deploy updates to gh-pages branch

### PR DESCRIPTION
**Pull request by dianashk**

### CODE REVIEW

**Comment by trescube** [comment]
:shipit:

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: php
+php:
+  - 5.4
+env:
+  global:
+    secure: DP5nm2y1uts/niMSRp4YtG1M076B87ukT1cZhtxMwxpmBxDMW6nqPpdJuN29GC00fSkF7FiOFCQD0BnO0RX5Me5xK+Cya2Cbpx04dRpUkqu3e5MThSBxaXNqgonf2Go6ndmy3ZSxCu7SpE+PvVU6fGdLClTs1d5RSDdvs5EGsRs=
```
Comment (orangejulius) [inline .travis.yml]: err...doesn't this have to stay secret?

Comment (trescube) [inline .travis.yml]: is that the github token for someone here?

Comment (dianashk) [inline .travis.yml]: no, that's encrypted and needs to be added to the travis file as per their instructions: http://couscous.io/docs/travis.html ... unless I totally butchered it 😬

Comment (orangejulius) [inline .travis.yml]: oh cool. yup looks right!

**Comment by orangejulius** [comment]
LGTM

**Pull request merged**